pkg/util/ip/ip.go

```diff
@@ -22,11 +22,26 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/apis/controlplane/v1beta2"
 )
 
+type IPFamily string
+
 const (
 	v4BitLen = 8 * net.IPv4len
 	v6BitLen = 8 * net.IPv6len
+
+	V4Family IPFamily = "ipv4"
+	V6Family IPFamily = "ipv6"
 )
 
+var LocalhostMap = map[IPFamily]string{
```

Comment:
please use names "LocalhostAddrMap" and "UnspecifiedAddrMap"

Reply:
Updated.